.changelog/5630.bugfix.md

```diff
@@ -0,0 +1,6 @@
+go/runtime/txpool: Don't abort runtime if node is not synced yet
+
+If the node hasn't finished syncing, `checkTxBatch` previously
+caused the runtime to be aborted, even though it wasn't the
```

Comment:
Why should txs check wait for a round sync? And if the runtime really aborts, do we need to fix the runtime instead?

Reply:
No, the runtime is aborted erroneously -- it's not the runtime's fault, but the fault of the code in `checkTxBatch`, which this PR fixes!  The previous code initiates the runtime abort, not the runtime itself -- in fact, the runtime doesn't even receive anything, because the block isn't synced yet.